src/sidebar/test/cross-origin-rpc-test.js

```diff
@@ -1,34 +1,36 @@
+import EventEmitter from 'tiny-emitter';
+
 import crossOriginRPC from '../cross-origin-rpc';
 import { $imports } from '../cross-origin-rpc';
 
+class FakeWindow {
+  constructor() {
+    this.emitter = new EventEmitter();
+    this.addEventListener = this.emitter.on.bind(this.emitter);
+    this.removeEventListener = this.emitter.off.bind(this.emitter);
+  }
+}
```

Comment:
This is fine. One thing I learned this morning that in some modern browsers it is actually possible to directly create instances of the browser's equivalent of `EventEmitter`, [EventTarget](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/EventTarget).

Reply:
good to know